lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -704,6 +705,9 @@ fromFlatSlot n = SlotId e (fromIntegral s)
 epochLength :: Integral a => a
 epochLength = 21600
 
+newtype SlotLength = SlotLength (Quantity "second/slot" Word8)
```

Comment:
I don't think inventing a double newtype wrapper helps. What about NominalDiffTime from the time library?

Reply:
it is already the standard I try to comply with. For example https://github.com/input-output-hk/cardano-wallet/blob/master/lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs#L362

Reply:
@rvl, I will use NominalDiffTime in the end

Reply:
from 1.9.1 in time there is `secondsToNominalDiffTime`, but we can use only 1.8.0.4 ;-/